server/emulator.go

```diff
@@ -62,6 +62,9 @@ func NewEmulatorAPIServer(server *EmulatorServer, backend *backend.Backend, stor
 
 	router.HandleFunc("/emulator/storages/{address}", r.Storage)
 
+	config := NewConfigInfo(server)
+	router.Handle("/emulator/config", config)
```

Comment:
maybe just basic `HandleFunc` ? and a new function there, instead new file `config.go` ?

Reply:
+1

Reply:
I would add to admin.go as we have those gathered there